main.py

```diff
@@ -58,3 +59,8 @@ def no_gameplan_exception():
         # main game loop
         bot.ingame_loop()
 
+Thread(target=main, daemon=True).start()
```

Comment:
This works a bit wonky, if the main function raises an exception the program wont exit/crasch because the failsafe loop is still active